docs/migrations/migrating-to-v5.md

```diff
@@ -109,6 +110,26 @@ The error message will be something like this:
 Uncaught Error: Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.
 ```
 
+To fix it, `useShallow` should help return a stable reference.
+
+```js
+// v5
+import { useShallow } from 'zustand/shallow'
+
+const [searchValue, setSearchValue] = useStore(
+  useShallow((state) => [state.searchValue, state.setSearchValue]),
+)
+```
+
+Here's another example that may cause infinite loops.
```

Comment:
I don't really understand why there is "another example," but the comment says that it's v4. Didn't you mean to say "Here's a similar example in v4 that may cause infinite loops?"

Reply:
I mean there are several cases that cause infinite loops.
I want to show typical two examples that are actually reported by users.

Reply:
Feel free to make suggestions to clarify that.